examples/java/build.gradle

```diff
@@ -56,6 +56,7 @@ dependencies {
   compile library.java.vendored_guava_26_0_jre
   compile library.java.kafka_clients
   compile project(path: ":sdks:java:core", configuration: "shadow")
+  compile project(path: ":runners:direct-java", configuration: "shadow")
```

Comment:
Took a closer look at this. It looks like the dependency is necessary because `DirectOptions` is referenced in the new test. Could you make it a `testCompile` dependency?